backbone.js

```diff
@@ -543,6 +543,18 @@
       return this.set(attr, void 0, _.extend({}, options, {unset: true}));
     },
 
+    // accepts an attribute name (or array of attribute names) and a transformer
+    // function which maps the current attribute value(s) to new attribute value(s)
+    update: function(attrName, transformer, setOptions) {
+      var attrNames = _.isArray(attrName) ? attrName : [attrName];
+      var newValues = _.reduce(attrNames, function(valueCollector, attr) {
```

Comment:
Your expectation from lodash users it to overwrite this function just to change it to 
`var newValues = _.reduce(attrNames, _.bind(function(valueCollector, attr) {`?
What will probably happen is someone will open a PR to change it to use bind. 
I figure it'd make sense to do it now.

I think https://github.com/jashkenas/backbone/pull/4083#issuecomment-252236917 is in agreement with my suggestion.